RecoParticleFlow/PFClusterProducerAlpaka/plugins/BuildFile.xml

```diff
@@ -0,0 +1,35 @@
+<library name="RecoParticleFlowPFClusterProducersPlugins" file="*.cc">
+  <use name="FWCore/Framework"/>
+  <use name="FWCore/ParameterSet"/>
+  <use name="FWCore/Utilities"/>
+  <use name="alpaka"/>
+  <use name="HeterogeneousCore/AlpakaTest"/>
```

Comment:
I hope dependence on `AlpakaTest` can be removed.

Reply:
This was removed.